composer.json

```diff
@@ -1,23 +1,29 @@
 {
-    "name": ":vendor_slug/:package_slug",
-    "description": ":package_description",
+    "name": "astrogoat/blog",
+    "description": "A Blog app for Strata",
     "keywords": [
-        ":vendor_name",
+        "astrogoat",
         "laravel",
-        ":package_slug"
+        "blog"
     ],
-    "homepage": "https://github.com/:vendor_slug/:package_slug",
+    "homepage": "https://github.com/astrogoat/blog",
     "license": "MIT",
     "authors": [
         {
-            "name": ":author_name",
-            "email": "author@domain.com",
+            "name": "Peter",
+            "email": "oniicoder@gmail.com",
             "role": "Developer"
         }
     ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.hlx.dev"
+        }
+    ],
     "require": {
         "php": "^8.0",
-        "astrogoat/strata": "^0.1.0",
+        "astrogoat/strata": "0.1.0",
```

Comment:
This should keep the tilde (`^`) otherwise it it will be locked to `0.1.0` instead of `0.1.0` and above.